modules/hamronization/summarize/main.nf

```diff
@@ -0,0 +1,38 @@
+process HAMRONIZATION_SUMMARIZE {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::hamronization=1.0.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hamronization:1.0.3--py_0':
+        'quay.io/biocontainers/hamronization:1.0.3--py_0' }"
+
+    input:
+    path(reports)
+    val(format)
+
+    output:
+    path("hamronization_combined_report.json"), optional: true, emit: json
+    path("hamronization_combined_report.tsv") , optional: true, emit: tsv
+    path("hamronization_combined_report.html"), optional: true, emit: html
+    path "versions.yml"                       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def outformat = format == 'interactive' ? 'html' : format
+    """
+    hamronize \\
+        summarize \\
+        ${reports.join(' ')} \\
```

Comment:
Does `$reports` (without join) work here? (honestly just curious!)

Reply:
Actually it does! I'll leave it like this as it works and I have two approvals but we can always remove it in the future if it's unnecesary or cause problems :+1:

Reply:
Awesome! Thanks for checking, can add it to the "Today I learned" book